payment_manual/indico_payment_manual/templates/transaction_details.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'events/payment/transaction_details.html' %}
+{% from 'message_box.html' import message_box %}
+
+{% block details %}
+    {% if transaction.status.name == 'pending' %}
+        <dt>{% trans %}Status{% endtrans %}</dt>
+        <dd>{% trans %}Awaiting bank transfer{% endtrans %}</dd>
+        <dt>&nbsp;</dt>
+        <dd>
+            <a class="i-button"
+               href="{{ url_for('plugin_payment_manual.received', transaction.registration.locator.uuid) }}">
```

Comment:
With the change to the RH inheritance suggested above, use `transaction.registration`. That will use the registration ID which is more appropriate in the management area.